### PR TITLE
Add thumb position on storefront

### PR DESCRIPTION
**Pull request by theomoura**
#### What is the purpose of this pull request? #### What problem is this solving?

Add the customization to the thumbnails image position on the storefront.

#### How should this be manually tested?

[Access](https://imageposition--storecomponents.myvtex.com/admin/cms/storefront); enter on a product with thumbnails; Go to product details section and change "Thumbnails position"

#### Types of changes

* [ ] Bug fix (a non-breaking change which fixes an issue)
* [x] New feature (a non-breaking change which adds functionality)
* [ ] Breaking change (fix or feature that would cause existing functionality to change)
* [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

**Review by rerissondaniel — CHANGES_REQUESTED** [review]
Missing CHANGELOG.md
Also, not directly related to this PR, but should there be a `below` option, as described [here](https://github.com/vtex-apps/store-components/issues/359).

**Review by rerissondaniel — APPROVED** [review]

**Review by JuanAlmeida12 — APPROVED** [review]

File: react/utils/thumbnailPositionEnum.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    name: 'editor.product-details.thumbnailsPosition.left',
+    value: 'left',
+  },
+  DISPLAY_RIGHT: {
+    name: 'editor.product-details.thumbnailsPosition.right',
+    value: 'right',
+  },
+}
+
+export function getThumbnailsPositionNames() {
```
Comment (matheusps) [inline react/utils/thumbnailPositionEnum.js:14]: You could do like:
```suggestion
export const getThumbnailsPositionNames = () =>
```

File: react/utils/thumbnailPositionEnum.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  return map(opt => opt.name, values(thumbnailsPosition))
+}
+
+export function getThumbnailsPositionValues() {
```
Comment (matheusps) [inline react/utils/thumbnailPositionEnum.js:18]: Here too 🤓

**Review by matheusps — APPROVED** [review]
Just personal preferences, change it if you feel like it 😬

**Review by ArthurSampaio — APPROVED** [review]

**Pull request merged**